File: back/WebApplication1/Models/Car.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace WebApplication1.Models
{
    public class Car
    {
        public long Id { get; set; }
        public string Description { get; set; }
        public string ModelOfCar { get; set; }
        public int TypeOfCar { get; set; }
        public int NumberOfSeats { get; set; }
        public double Price { get; set; } //per day
        public double Rating { get; set; }
        public string ImagePic { get; set; }
        public string NameOfCompany { get; set; }
        public bool IsReserved { get; set; }
    }
}

File: front/my-app/src/app/entities/Car.ts
export class Car {
    constructor(
      public id: number,
      public description: string,
      public modelofcar: string,
      public typeofcar: number,
      public numberofseats: number, //passengers
      public price: number,
      public rating: number,
      public imagepic: string,
      public nameOfcompany: string,
      public isReserved: boolean
    ) {}
  }